pep-0001.txt

```diff
@@ -457,19 +457,31 @@ if they are never meant to be completed.  E.g. :pep:`1` (this PEP).
 PEP Maintenance
 ---------------
 
-In general, Standards track PEPs are no longer modified after they have
-reached the Final state. Once a PEP has been completed, the Language and
-Standard Library References become the formal documentation of the expected
-behavior.
+In general, PEPs are no longer substantially modified after they have reached
+the Accepted, Final, Rejected or Superseded state. Once resolution is reached,
+a PEP is considered a historical document rather than a living specification.
+Formal documentation of the expected behavior should be maintained elsewhere,
+such as the `Language Reference`_ for core features, the `Library Reference`_
+for standard library modules or the `PyPA Specifications`_ for packaging.
 
 If changes based on implementation experience and user feedback are made to
-Standards track PEPs while in the Accepted or Provisional State, those changes
-should be noted in the PEP, such that the PEP accurately describes the state of
+Standards track PEPs while in the Provisional or (with SC approval) Accepted
+state, they should be noted in the PEP, such that the PEP accurately describes
 the implementation at the point where it is marked Final.
 
-Informational and Process PEPs may be updated over time to reflect changes
-to development practices and other details. The precise process followed in
-these cases will depend on the nature and purpose of the PEP being updated.
+Active Informational and Process PEPs may be updated over time to reflect
+changes to development practices and other details. The precise process
+followed in these cases will depend on the nature and purpose of the PEP
+being updated. Substantive changes to governance PEPs must be reviewed by the
+Steering Council (by opening a `Steering Council issue`_).
```

Comment:
Again, is this captured elsewhere to your satisfaction? If so the sentence starting "Substantive" can be removed.

Reply:
I did some searching (before and now) and I couldn't find anywhere else that explicitly mentioned this. With how long PEP 1 is, it is not impossible that I missed it, but at least it didn't appear to be anywhere I could think to find it, and this seems to be the most logical place to put it.

We could also modify this to say

```suggestion
being updated. Substantive changes to Active Process PEPs must be
reviewed by the Steering Council (by opening a `Steering Council issue`_).
```

since that more clearly defines what PEPs must have this approval, since "governance PEPs" are not so well defined.

To note, as I've noticed previously, PEP 13 is inexplicably listed as an Informational PEP rather than a Process PEP, despite it being the Meta/Process PEP of all Process PEPs as it defines the "process" by which the Python community governs itself and in turn decides on the PEP process, and is thus the most immutable of them all.

Reply:
Your suggested wording looks good (although, I would actually interpret "governance PEPs" as just 13 and the 8000 series, so perhaps it would be useful to check what the Council care about enough to add to their workload!

A

Reply:
Any thoughts on this @brettcannon ?

Reply:
The SC *cannot* unilaterally change PEP 13. The procedure to change PEP 13 is described [in the PEP itself](https://peps.python.org/pep-0013/#changing-this-document). The same goes for the [PyPA governance PEP](https://peps.python.org/pep-0609/#updates-to-the-governance-specification-processes) for example.
IMO the previous wording is enough here, no addition about the SC is necessary.

Reply:
Right, but this is about Active Process PEPs in general, not anything specific to one PEP, which is what my suggestion above changes it to be more clear on (and, inexplicably, PEP 13 is listed as a mere _"Informational"_ PEP rather than a "Process" PEP, so technically it wouldn't apply to PEP 13 at all, albeit almost certainly due to an oversight). That said, the previous sentence says "The precise process followed in these cases will depend on the nature and purpose of the PEP being updated.", so if there's no need to explicitly specify this here, I'll just remove it.

```suggestion
in question.
```

To note, I originally included another clause specifying that beyond SC review, any guidance in the PEP in question [e.g. PEP 13, 609, etc] also needed to be followed when updating it, but I had elided it in the current version for brevity, due to concerns about the original proposed wording being too verbose and overly specific.